pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -133,6 +133,8 @@ func newCertRotationController(
 		certrotation.RotatedSigningCASecret{
 			Namespace:              operatorclient.OperatorNamespace,
 			Name:                   "aggregator-client-signer",
+			JiraComponent:          "kube-apiserver",
+			Description:            "Signer for the kube-apiserver to create client certificates for aggregated apiservers to recognize as a front-proxy.",
```

Comment:
pull descriptions to a separate PR, just do owners here.

In the descriptions PR, add information about which processes and flags these are attached to.  Or in cases where its a constituent indicate that it is combined with other certs to eventually be passed to kube-apiserver --foo.